test/decl/inherit/inherit.swift

```diff
@@ -30,13 +30,13 @@ class D3 : Any, A { } // expected-error{{superclass 'A' must appear first in the
 class D4 : P & P1, A { } // expected-error{{superclass 'A' must appear first in the inheritance clause}}{{18-21=}}{{12-12=A, }}
 
 // Struct inheriting a class
-struct S : A { } // expected-error{{inheritance from non-protocol type 'A'}}
+struct S : A { } // expected-error{{struct type cannot inherit from non-protocol type 'A'}}
 
 // Protocol inheriting a class
 protocol Q : A { }
 
 // Extension inheriting a class
-extension C : A { } // expected-error{{inheritance from non-protocol type 'A'}}
+extension C : A { } // expected-error{{extension type cannot inherit from non-protocol type 'A'}}
```

Comment:
I think that there is no such term as `extension type`, to me this should be the same diagnostic as `class C: A {}` `class type cannot inherit ...`